thymeleaf-spring6/src/main/java/org/thymeleaf/spring5/ISpringTemplateEngine.java

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
I understand there are still changes to be made to packages here `.thymeleaf.spring5.` -> `.thymeleaf.spring6.` -> and also imports on `org.thymeleaf.spring5.*` classes, right?

Reply:
Correct.